.github/workflows/ci.yml

```diff
@@ -4,9 +4,9 @@ name: CI
 # events but only for the master branch
 on:
   push:
-    branches: [ main ]
+    branches: [ main, develop ]
   pull_request:
-    branches: [ main ]
+    branches: [ main, develop ]
```

Comment:
Your #264 PR seems to be mixed in with this one. Do you wanna separate them out?

Reply:
I went ahead and removed the devleop branch stuff myself.